src/tests/issues.targets

```
@@ -694,6 +694,9 @@
 
     <!-- NativeAOT specific -->
     <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TestBuildMode)' == 'nativeaot' and '$(RuntimeFlavor)' == 'coreclr'">
+        <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/StaticVirtualMethods/RegressionTests/GetInterfaceMapWithStaticVirtualsAndConstraints/**">
+            <Issue>https://github.com/dotnet/runtime/issues/89122</Issue>
```

We can track this in the "`GetInterfaceMap` doesn't really work" uber issue. There's already a bunch of tests blocked on it that we can ignore forever (or fix if there's ever a good enough business justification).

```suggestion
            <Issue>https://github.com/dotnet/runtime/issues/89157</Issue>
```